### PR TITLE
UIOR-690 Populate linked instances accordion with instance details

### PR DESCRIPTION
**Pull request by NikitaSedyx**
## Purpose

https://issues.folio.org/browse/UIOR-690

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [x] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/components/POLine/LineLinkedInstances/LineLinkedInstances.test.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+}), { virtual: true });
+
+const renderLineLinkedInstances = ({ line = {}, toggleSection = jest.fn() }) => (render(
+  <MemoryRouter>
+    <LineLinkedInstances
+      line={line}
+      toggleSection={toggleSection}
+    />
+  </MemoryRouter>,
+));
```
Comment (axelhunn) [inline src/components/POLine/LineLinkedInstances/LineLinkedInstances.test.js:26]:
```suggestion
const renderLineLinkedInstances = ({ line = {}, toggleSection = jest.fn() }) => render(
  <LineLinkedInstances
    line={line}
    toggleSection={toggleSection}
  />,
  { wrapper: MemoryRouter },
);
```

File: src/components/POLine/LineLinkedInstances/formatters.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    .map(({ name }) => name)
+    .join('; ');
+};
+
+export const formatRelations = (parents = [], children = [], relationTypes = []) => {
+  const relationTypesMap = relationTypes.reduce((acc, rType) => {
+    acc[rType.id] = rType.name;
+
+    return acc;
+  }, {});
```
Comment (axelhunn) [inline src/components/POLine/LineLinkedInstances/formatters.js:24]:
```suggestion
  const relationTypesMap = Object.fromEntries(relationTypes.map(rt => [rt.id, rt.name]);
```

**Review by AndreiShumski2 — APPROVED** [review]

File: src/components/POLine/LineLinkedInstances/useLinkedInstances.test.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+describe('useLinkedInstances', () => {
+  beforeEach(() => {
+    useOkapiKy
+      .mockClear()
+      .mockReturnValue({
+        get: (path) => ({
+          json: () => kyResponseMap[path],
+        }),
+      });
```
Comment (axelhunn) [inline src/components/POLine/LineLinkedInstances/useLinkedInstances.test.js:52]: is it OK returning not a Promise?

**Review by axelhunn — APPROVED** [review]

File: src/components/POLine/LineLinkedInstances/useLinkedInstances.test.js (hunk 3 above)
Comment (NikitaSedyx) [reply src/components/POLine/LineLinkedInstances/useLinkedInstances.test.js:52]: yes

**Review by uladislausamets — APPROVED** [review]

File: src/components/POLine/LineLinkedInstances/formatters.js (hunk 2 above)
Comment (NikitaSedyx) [reply src/components/POLine/LineLinkedInstances/formatters.js:24]: I'm not worry about performance, but reduce approach has less complexity + browser support is better (not sure we transpile fromEntries - probably not our case)

**Pull request merged**